framework/core/src/Mail/Mailer.php

```diff
@@ -21,7 +21,7 @@ public function __construct(
         string $name,
         Factory $views,
         TransportInterface $transport,
-        Dispatcher $events = null,
+        ?Dispatcher $events,
```

Comment:
Removing the default `null` value will increase the required code outside of this class, is there a specific reason to do this?

Reply:
@luceos dependency injection fills the value

Reply:
there is a non optional argument after it, so it being optional or not makes no difference in this case. it can't be optional, only nullable